script/QA/QA_20.txt

```diff
@@ -196,95 +211,111 @@ That being said,
 [sha:474e72fffe9c3e1eb416ff37e8cd459487eadf8d]{
 -- Page 187, Offset 43662.
 -- ちょっと気になる事があるんだけどぉ。
-there is something that's been bothering me.
+ there's something that's%{n}
+been bothering me.
 }
 [sha:f458e67001baefbd8781aef749f1f319ecdc0498]{
 -- Page 188, Offset 43663.
 -- 首を斬らない方を選んだ後、
-After choosing not to cut off her head,
+If you choose not to cut off her head,
 }
 [sha:cbee2c0c57892acfe87aa805b7faac4b3f7f73c7]{
 -- Page 188, Offset 43664.
 -- プレイによってノエルの反応、変わってニャい？
-doesn't Noel's reaction change depending on the play nya?
+ doesn't Noel's%{n}
+reaction depend on how you played?
 }
 [sha:1d79953a4633229ae088fa396e3cec759aa9a80c]{
 -- Page 189, Offset 43665.
 -- ノエルとの親密度が高いと、
-If you reach a high level of intimacy with Noel,
+If you've reached a high level of intimacy with Noel,
```

Comment:
people usually use "affection" to talk about dating game points, but I don't really think it matters...